modules/nw-kuryr-cleanup.adoc

```diff
@@ -0,0 +1,280 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-cleanup_{context}"]
+= Cleaning up resources after migration
+
+After migration from the Kuryr network plugin to the OVN-Kubernetes network plugin completes, clean up the resources that Kuryr created previously. 
+
+.Prerequisites
+
+* You installed the {product-title} CLI, `oc`.
+* You installed a Python interpreter.
+* You installed the `openstacksdk` Python package.
+* You installed the `openstack` CLI.
+* You have access to the underlying {rh-openstack} cloud.
+* You can access the cluster as a user with the `cluster-admin` role.
+
+[NOTE]
+====
+At the time of writing when OCP16 is used, it includes ancient version of
+`openstacksdk` package, that does not support tags for Octavia objects. It is
+recommended to use either workstation with modern operating system with newer
+version of `openstacksdk` package or `virtualenv` and install appropriate
+version from link:https://pypi.org[pypi].
+====
+
+.Setup virtualenv
```

Comment:
This added virtualenv content should either be a new module or an optional step 1 in the procedure that follows it. Would you like me to do it? @gryf

Reply:
I guess optional step will serve here better than separate module.

Reply:
@gryf Noticed this isn't broken out yet. My inclination is to remove it unless it is absolutely necessary. We typically don't include "if you decide" sorts of information in a procedure. 

If it is necessary, it should become the first few steps of the `.Procedure`.

Reply:
I think it is good to have the virtualenv.
In my case, during the verification of this procedure, the 16.2 [latest cdn](http://rhos-qe-mirror-tlv.usersys.redhat.com/rcm-guest/puddles/OpenStack/16.2-RHEL-8/latest_cdn/COMPOSE_ID) didn't include a proper openstacksdk version, so a virtualenv with the correct Packages was needed.

[0] https://redhat-internal.slack.com/archives/CH98TDJUD/p1679991410225769?thread_ts=1679919862.338489&cid=CH98TDJUD

Reply:
@imatza-rh @gryf In that case, my preference is to require venv for this procedure. 

Have you run this part of the procedure past Support? They're typically not keen on requiring much package installation. :\

Reply:
FYI @sferich888 -- is what's proposed in L20-L63 okay? Can't seem to find prior uses of venv in the docs, and I assume there might be a reason for it.

Reply:
Its fine. However all this is really doing is installing the `openstack` command/cli (and ensuring that you can run it); I think explaining that better in the prerequisites is key to understanding that by putting you in a virtual environment; the version/packages we install here don't conflict with system packages or other python packages (that you may be using). 

virtual environment are a best practices (and I recommend using one here); but not strictly required.